app/src/main/java/org/schabi/newpipe/player/Player.java

```diff
@@ -3056,6 +3060,42 @@ public void selectQueueItem(final PlayQueueItem item) {
         playQueue.setIndex(index);
     }
 
+    private void heldQueueItem(final PlayQueueItem item, final View view) {
```

Comment:
Create a helper class `org.schabi.newpipe.player.QueueItemMenuUtil` with a method `openPopupMenu(final PlayQueueItem item, final View view, final boolean hideDetails)` and use it in both `Player` and `PlayQueueActivity`.

Reply:
If I'm not mistaken it'll need the fragment manager and context as parameters too?

Reply:
Yes, you are right

Reply:
Done in https://github.com/TeamNewPipe/NewPipe/pull/6955/commits/a1cb3e59d6c5babcb79a6bedfc043494330b4cac.

Turns out I had to have 7 parameters:
- `playQueue`
- `item`
- `view`
- `hideDetails`
- `fragmentManager`
- `context`
- `tag` <- The PlaylistAppendDialog and PlaylistCreationDialog needs the class's `TAG`.